cypress/integration/core-opensearch-dashboards/opensearch-dashboards/apps/query_enhancements/helpers.js

```diff
@@ -0,0 +1,342 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/**
+ * Get specific row of DocTable.
+ * @param {number} rowNumber Integer starts from 0 for the first row
+ */
+export function getDocTableRow(rowNumber) {
```

Comment:
let me move this helpers to utils/apps/data_explorer/commands.js and move query specific to utils/apps/query_enhancements/commands.js

Reply:
if the utilities are too many, we can create sub-files. like utils/apps/data_explorer/doc_table/commands.js and utils/apps/data_explorer/side_panel/commands.js